src/core/client/stream/tabs/Live/LiveCommentEnteredSubscription.tsx

```diff
@@ -13,13 +13,26 @@ import { LiveCommentEnteredSubscription } from "coral-stream/__generated__/LiveC
 
 import { isPublished } from "../shared/helpers";
 import handleNewCommentInStory from "./helpers/handleNewCommentInStory";
+import handleNewReplyInConversation from "./helpers/handleNewReplyInConversation";
 
-function liveInsertionEnabled(environment: Environment): boolean {
+function liveMainStreamInsertionEnabled(environment: Environment): boolean {
   const liveChat = lookup(environment, LOCAL_ID).liveChat;
 
   return liveChat.tailing;
 }
 
+function liveConversationInsertionEnabled(environment: Environment): boolean {
+  const liveChat = lookup(environment, LOCAL_ID).liveChat;
+
+  return liveChat.tailingConversation;
+}
+
+function conversationRootCommentID(environment: Environment): string | null {
+  const liveChat = lookup(environment, LOCAL_ID).liveChat;
+
+  return liveChat.conversationRootCommentID;
+}
+
```

Comment:
What do you think about calling these `isLiveMainStreamInsertionEnabled`, `isLiveConversationInsertionEnabled`, `getConversationRootCommentID` instead? I mistook them for variables on first glance instead of functions.